operator/api/v1alpha1/observatorium_types.go

```diff
@@ -64,6 +64,26 @@ type ObjectStorageConfigSpec struct {
 	Key string `json:"key"`
 }
 
+type LokiObjectStorageConfigSpec struct {
+	// Object Store Config Secret Name
+	Name string `json:"name"`
```

Comment:
Could we use `SecretName`? I think this helps communicate the expectation of the field, vs maybe suggesting we are naming this struct

Reply:
Will do, but I will leave the thanos config as is.